### ISSUE TITLE
Priority system for new players?

### ISSUE THREAD

**Issue by Surobutox** [EarthMC/Issue-Tracker#1008]
My idea is that there is a 3 tier priority, it goes 1 - buyers of ranks 2 -first time players 3 - regulars. This will make it so towns increase in residents, there will be more votes a day, and therell be more regulars after actually trying the server rather than getting on the queue and leaving immediatly.

We need more players to expand towns and this will benefit everyone, votes for you, players for us and you.

**Comment by FutureLemon** [comment]
I like this idea but honestly I also feel like is endorses pay to play.

**Comment by jkmartindale** [comment]
> I like this idea but honestly I also feel like is endorses pay to play.

Isn't the only change being proposed is first-time players getting priority over non-priority players? The queue at present is already pay to play.

**Comment by Surobutox** [comment]
Yeah, only first time players get priority, it would be beneficial


Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Comment by Mineturtle795** [comment]
Yeah, really beneficial for ggovi and his bots.

**Comment by Surobutox** [comment]
That is a fair point

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by Surobutox** [comment]
Its up for developers to do tho

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by MineHero43** [comment]
Actually I like this idea, it means that new player's first impression of joining emc isn't having to wait for hours in a queue. Prioritizing new players means they'll be able to get a feel for the server much sooner and then if they like what they see they can purchase a rank. Many new players are immediately turned off by the long waiting time and I would assume that's an issue for the server.

The only downside I can see is this being abused with bot accounts though

**Comment by jkmartindale** [comment]
> Yeah, really beneficial for ggovi and his bots.

He tends to come online when the server queue is short or empty though, so I'm not sure if it would make a huge difference.

**Comment by Mineturtle795** [comment]
Yes but ggovi could just get 150 bots into the queue, which would all have priority and make it impossible for any non-prio people to join, which is counterintuitive.

**Comment by Surobutox** [comment]
> Yeah, really beneficial for ggovi and his bots.
> 
> He tends to come online when the server queue is short or empty though, so I'm not sure if it would make a huge difference.

but then he'd be able to bot whenever so yeah

**Comment by FutureLemon** [comment]
> Yeah, really beneficial for ggovi and his bots.

You can easily counter most bots if not all bots if the server gets some sort of captcha you have to type in. I have seen it in the past and it seems like it works. We could also re-introduce the compass because currently you instantly get put in queue without any effort and that at its own already creates many problems.

**Comment by Prismary** [comment]
I know this issue is very old at this point, but I'd just like to say that in my opinion this is a terrible idea. Before mostly switching to EMC I used to play 2b2t a lot, which famously also has to deal with an hour-long queue. And whenever a player influx happened due to a popular youtuber publishing a video or something similar, the queue would be clogged with new players that would log in once, maybe twice, and then never again. This development was always devastating for the current active community that was no longer able to play in the same capacity they used to. They're the ones that made the server interesting in the first place - not the players joining once to take a look. The first time this happened the admin even introduced a "veteran queue" which prioritized old players over ones that joined past June 1st, 2016 in order to prevent the community from falling apart entirely.   

tl;dr: Prioritizing new players may help towns grow in the short term but drives away the active community. Not a trade-off worth considering in my opinion.

**Comment by Basically-Andrew** [comment]
A new queuing system has been introduced that allows more players of lower rank into Towny.
